ofl/bethellen/METADATA.pb

```diff
@@ -0,0 +1,16 @@
+name: "Beth Ellen"
+designer: "Rob Jelinski"
+license: "OFL"
+category: "HANDWRITING"
+date_added: "2019-03-06"
+fonts {
+  name: "Beth Ellen"
+  style: "normal"
+  weight: 400
+  filename: "BethEllen-Regular.ttf"
+  post_script_name: "BethEllen-Regular"
+  full_name: "Beth Ellen Regular"
+  copyright: "Copyright 2018 The Beth Ellen Project Authors (https://github.com/felipesanches/BethEllen)"
+}
+subsets: "latin"
```

Comment:
Did you confirm it has 100% of the "Google Latin Core" glyph set?

Reply:
No, I haven't. I'll do it.

Reply:
I have just reviewed it and there's a whole bunch of glyphs from that set still missing.

Reply:
For completeness, here's the glyph set we're talking about: https://github.com/googlefonts/gftools/blob/e58fa6f54d662c237f47b39c2feb2017cb53689f/Lib/gftools/encodings/GF%20Glyph%20Sets/GF-latin-core_unique-glyphs.nam